st2actions/st2actions/worker.py

```diff
@@ -83,26 +84,32 @@ def execute_action(self, liveaction):
             LOG.exception('Failed to find liveaction %s in the database.',
                           liveaction.id)
             raise
+        # stamp liveaction with process_info
+        runner_info = system_info.get_process_info()
```

Comment:
As we talked since logs can be routed through syslog we will have to do it from outside of st2.